core/scss/components/organisms/_global-footer.scss

```diff
@@ -0,0 +1,141 @@
+@charset "UTF-8";
+
+//
+// Global Footer
+//
+// Global footer element as specified by the Identity Guidelines.
+//
+// Markup: ../../templates/components/organisms/global-footer/global-footer.twig
+//
+// Style guide: Organisms.Global Footer
+//
+
+// Global styles for the global footer.
+.global-footer {
+
+  @include centered-grid-container;
+  background-color: $color-cardinal-red;
+  color: $color-white;
+  padding: 2.5rem 0;
```

Comment:
Effort is the same in writing, so I am open to arguments against, but I am a fan of using mixins where and when we can. This could be `@include padding(2.5rem 0);`